src/cfclient.app.src

```diff
@@ -4,10 +4,10 @@
   [
     {description, "Harness Feature Flags Server SDK"},
     {pkg_name, "harness_ff_erlang_server_sdk"},
-    {vsn, "2.0.0"},
+    {vsn, "2.0.1"},
     {registered, []},
     {mod, {cfclient_app, []}},
-    {applications, [kernel, stdlib, cfapi]},
+    {applications, [kernel, stdlib, cfapi, base64url, murmur, jsx, mochiweb]},
```

Comment:
This is the fix for the missing deps. In Erlang, you have "Applications" and "libraries" - applications start a process, and libraries are just. This `application` option as far as all documentation suggests is for applications which start a process and not libraries. However - these deps were missing from Elixir releases (Erlang releases were fine) adding them here ensures Elixir releases contain them